### ISSUE TITLE
setvideomode: second try with videoantialiasing=0 fails

### ISSUE THREAD

**Issue by FrMo** [gak/gravit#66]
gravit fails when trying to open the visual and antialiasing is not supported:

> Detected 32 BPP
> SDL_SetVideoMode failed: Couldn't find matching GLX visual
> You have videoantialiasing on. I'm turning it off and restarting...
> SDL_SetVideoMode failed: Couldn't find matching GLX visual
> There has been an error on start-up. Read your gravit configration file to possibly fix this.

when I edit gravit.cfg and set `videoantialiasing 0` manually, everything is fine again.

(another bug related to my corei5 "build-in" graphics on linux)

**Comment by gak** [comment]
I've seen that happen too, with the exact resolution.

**Comment by FrMo** [comment]
Possible reason: the code in gfxSetResolution does not reset the FSAA attributes when called with video.screenAA=0 (else branch missing). A solution could be:

``` c
    if (video.screenAA) {
        SDL_GL_SetAttribute( SDL_GL_MULTISAMPLEBUFFERS, 1);
        SDL_GL_SetAttribute( SDL_GL_MULTISAMPLESAMPLES, 4);
    } else {
        SDL_GL_SetAttribute( SDL_GL_MULTISAMPLEBUFFERS, 0);
        SDL_GL_SetAttribute( SDL_GL_MULTISAMPLESAMPLES, 0);
    }
```

additionally, we could check other values for SDL_GL_MULTISAMPLESAMPLES: 1,2,4,8 seem to be common values.

**Comment by FrMo** [comment]
problem is fixed in gravit 0.5.1.
